python/llm/setup.py

```diff
@@ -45,36 +45,63 @@ def get_llm_packages():
     return llm_packages
 
 
-lib_urls = [
+lib_urls = {}
+lib_urls["Windows"] = [
     "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/llama.dll",
     "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/quantize-llama.exe",
     "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/gptneox.dll",
     "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/quantize-gptneox.exe",
     # TODO: add bloomz
 ]
+lib_urls["Linux"] = [
+    "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/libllama_avx2.so",
+    "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/libllama_avx512.so",
+    "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/quantize-llama",
+    "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/libgptneox_avx2.so",
+    "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/libgptneox_avx512.so",
+    # "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/quantize-gptneox-avx2",
+    # "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/quantize-gptneox-avx512",
+]
```

Comment:
current build guide requires a strict order between code block 49-55 and code block 56-64. We could merge this for now but I am not sure it a good way for future developing.

Reply:
sed -> pass platform args.